### PR TITLE
Reduce number of nature reserve POIs at zoom 10.

### PR DESCRIPTION
**Pull request by zerebubuth**
Use same tier 2 zoom range for nature reserve POIs as the landuse polygon features.

Connects to #1749.

### CODE REVIEW

**Comment by nvkelso** [comment]
This should be consolidated for general fix in https://github.com/tilezen/vector-datasource/issues/1609.

**Comment by zerebubuth** [comment]
I added some stuff to deal with #1609.

File: yaml/pois.yaml [hunk 1]
```diff
@@ -951,15 +965,6 @@ filters:
     output:
       <<: *output_properties
       kind: battlefield
-  # park w/ US Parks Service operator
```
Comment (nvkelso) [inline yaml/pois.yaml]: Was this a catch-all for any leisure or landuse park before, but with a bad comment?

Comment (nvkelso) [reply yaml/pois.yaml]: Yes, on line 432 in this diff.

**Pull request merged**